Iteam1337/brevduvor
# refactor: locale context

**Issue:** Closes #38 
**Intent:**

To allow a user to switch language, this adds a context which holds the current language selected and then each component which has user-facing string consumes curried `toString`-functions with the language selected.


```ocaml
let ({errorToString, translationsToString}, _changeLocale): LocaleContext.t =
    LocaleContext.use();


<p> {errorToString(NoData)}->React.string </p>
```

**How to test (optional):**

Run the client, select another language in the front-page. The language should change.
